### PR TITLE
add using custom validator

### PR DESCRIPTION
**Pull request by flec**
Fixes https://github.com/PolymerElements/iron-checked-element-behavior/issues/18 by adding check for custom validator.

### CODE REVIEW

File: bower.json [hunk 1]
```diff
@@ -21,7 +21,8 @@
   "dependencies": {
     "polymer": "Polymer/polymer#^1.0.0",
     "iron-validatable-behavior": "PolymerElements/iron-validatable-behavior#^1.0.0",
-    "iron-form-element-behavior": "PolymerElements/iron-form-element-behavior#^1.0.0"
+    "iron-form-element-behavior": "PolymerElements/iron-form-element-behavior#^1.0.0",
+    "iron-validator-behavior": "PolymerElements/iron-validator-behavior#^1.0.1"
```
Comment (notwaldorf) [inline bower.json]: I think this might need to be a devDependency, since it's only used for testing. WDYT?

File: test/custom-validator.html [hunk 2]
```diff
@@ -0,0 +1,23 @@
+<!--
+@license
+Copyright (c) 2015 The Polymer Project Authors. All rights reserved.
```
Comment (notwaldorf) [inline test/custom-validator.html]: 2016, please :)

**Comment by notwaldorf** [comment]
Jus a couple minor comments, but this looks good. /cc @bicknellr in case he has any comments

**Comment by flec** [comment]
Thanks @notwaldorf. Updated as you suggested.

File: test/validation-checkbox.html [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+    <simple-checkbox id="checkbox" validator="customValidator"></simple-checkbox>
+  </template>
+</dom-module>
+<script>
+  Polymer({
+
+    is: 'validation-checkbox',
+
+    ready: function() {
+      this.$.validator.validate = this.validate.bind(this);
```
Comment (bicknellr) [inline test/validation-checkbox.html:27]: Why is `validate` here, rather than in the definition of `custom-validator`?

**Pull request closed without merging**